### ISSUE TITLE
README hotfix?

### ISSUE THREAD

**Issue by Endyl** [Pepita73/webproghu_dev#36]
Szerintem ráférne egy frissítés a README.md fájlra `master`-ben addig is, amíg nem lesz release, hogy a legkönyebben elérhető dokumentációs felületen is legyen valami aktuális információ.

Össze kéne szedni, hogy mi legyen benne.

Szerintem #34-nek ott lenne a helye.

Egyéb módosítási igényetek van?

**Comment by Pepita73** [comment]
Igen, maradjon benne a Container indítás és a Drupal telepítés rész, ami előttük van, az már jórészt elavult.
Esetleg pár még mindig hasznos parancs megmaradhat belőle.
Utána jöhetne a git setup, illetve a flow használata szerintem.

**Comment by Pepita73** [comment]
Szerintem a hotfix a címben megtévesztő. :)

**Comment by Endyl** [comment]
Hotfix, mert `master`-ba kell menjen 1.0.0 előtt, hogy githubon, vagy egy normál `clone` után egyből egy használható README-t kapjon mindenki :)

**Comment by Pepita73** [comment]
@Endyl , szükség van további válaszokra, vagy ezen lehet dolgozni?
(Szerintem jó lenne, ha elvinnéd. :) )
